packages/plugins/accordion/src/utils/index.ts

```diff
@@ -0,0 +1,7 @@
+import { ReactElement } from 'react';
```

Comment:
should be removed after adding svg as string

Reply:
fixed in latest commit